api/grpc_service.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package envoy.api.v2;
+
+import "api/data_source.proto";
+
+import "google/protobuf/duration.proto";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: gRPC service]
+// [#proto-status: draft]
+
+// gRPC service configuration. This is used by :ref:`ApiConfigSource
+// <envoy_api_msg_ApiConfigSource>` and filter configurations.
+message GrpcService {
+  oneof target_specifier {
+    // The name of the upstream gRPC cluster when using Envoy's gRPC client. SSL
```

Comment:
nit: s/SSL/TLS everywhere.

Reply:
We should keep it as `SslCredentials` to match the terminology used in the Google gRPC library for that specific message, but elsewhere +1.

Reply:
Sure sounds good.